src/core/Button/Button.md

```diff
@@ -16,6 +16,24 @@ import { Button } from 'suomifi-ui-components';
 </>;
 ```
 
+```js
+import { Button, Paragraph, Text } from 'suomifi-ui-components';
+
+<>
+  <Button
+    aria-describedby="additiona-info"
+    aria-disabled={true}
+    onClick={() => console.log('Test button click')}
+    type="submit"
+  >
+    Aria-disabled button
+  </Button>
+  <Paragraph id="additiona-info">
```

Comment:
Not a problem, but as this is a example we might want to fix the typo in the id;
From _additiona-info_ to _additiona**l**-info_

Reply:
I'd also like to apply a style here, but no comments from design yet.

Reply:
Typo fixed and text for additional-info improved.

Reply:
As agreed, styling this will be handled later on when PAMU team is again available.